components/choices/index.js

```diff
@@ -39,10 +39,10 @@ const updateChoiceById = async (req, response) => {
   let i = 0;
 
   try {
-    const values = getInputFromSigPayload(payloadBytes);
+    const {votesData, isXTZ} = getInputFromSigPayload(payloadBytes);
```

Comment:
Do we need to take the isXTZ here ? It's already set in the poll collection whether the poll is xtz based or not. we can take the data from there right ?

Reply:
Totally right. Change made!